modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -5,23 +5,37 @@ endif::[]
 :imagesdir: ../assets/images/
 :page-aliases: to-share-an-asset-with-a-user.adoc
 
-Exchange administrators and asset administrators can share an asset with another user in their organization, with the whole organization, or with an external organization.
+Exchange administrators and asset administrators can share an asset with another user or team in their organization, with the whole organization, or with an external organization.
 
 Any Exchange user can click the *Shared with me* link in the left navigation menu of Exchange to show all assets shared with their user account and all assets shared with their organization.
 
+To see who has access to an asset, open the asset portal page and click *Share*.
```

Comment:
The "Share" modal only shows identities (users/teams/organizations) that have access to the asset because they have asset-level permission assigned.
You will not see identities that have access to the asset because they have bg-level permission assigned. There is no easy way to see those users.

I would either
 - Call out that the list in the Share modal is not complete and explain you only see identities with asset-level permissions assigned.
 - Avoid this statement altogether to avoid confusion

Reply:
Thank you! Since there is no easy way to see everyone with access to an asset, I decided just to remove this comment.